packages/next/client/image.tsx

```diff
@@ -374,6 +378,18 @@ export default function Image({
         `Image with src "${src}" has both "priority" and "loading='lazy'" properties. Only one should be used.`
       )
     }
+    if (placeholder === 'blur') {
+      if ((widthInt || 0) * (heightInt || 0) < 1600) {
+        console.warn(
+          `Image with src "${src}" is smaller than 40x40. Consider removing the "placeholder='blur'" property to improve performance.`
```

Comment:
This wording might confuse some people as to whether it's talking about the image's intrinsic size or the size given to the component.
Maybe something like  "Image with src "${src}" is has size attributes smaller than 40x40".

Reply:
But what is `size`?

I think the distinction doesn't really matter because we never upscale images. So it can be either the user-defined `width` and `height` or it can be the original image is too small.